internal/querycoord/task.go

```diff
@@ -276,19 +275,27 @@ func (lct *loadCollectionTask) timestamp() Timestamp {
 }
 
 func (lct *loadCollectionTask) updateTaskProcess() {
```

Comment:
what is the difference. between postExecute and updateTaskProcess?
updateTaskProcess is suppoted to call multiple times?

Reply:
like show the percentage of load collection progress?